### DISCUSSION TITLE
Telegram publication

### DISCUSSION THREAD

**Discussion by sgorelova** [requarks/wiki#6474] [Help / Questions]
Hi everyone!

Is it possible to set up Telegram integration in order to automatically send updates and news from wiki.js to Telegram channel? I've found only [something similar](https://github.com/requarks/wiki/discussions/4431) regarding email.

**Comment by NGPixel** [comment]
There's already a channel you can subscribe to: https://t.me/wiki_js

**Comment by sgorelova** [comment]
@NGPixel, you didn't get it correctly - I mean integrating my work wiki.js knowledge base to our corporate tg channel in order to send wiki updates to this tg channel automatically.
